cpp/ql/test/experimental/query-tests/Security/CWE/CWE-1126/semmle/tests/test.c

```diff
@@ -0,0 +1,15 @@
+void workFunction_0(char *s) {
+  int intIndex = 10;
+  char buf[80];
+  while(intIndex > 2) // GOOD
+  {
+    buf[intIndex] = 1;
+    intIndex--;
+  }
+  while(intIndex > 2)
```

Comment:
Could we use a distinct variable for the second loop?  As it is, `intIndex` is already `2` following execution of the first loop so the second loop would not be entered.

Reply:
thank you for noticing this error.
but considering your request to increase the number of test cases, I propose to leave the variable in a single form.

Reply:
I'm happy with your fix.  :+1: